manifests/00-crd-operator-config.yaml

```diff
@@ -13,6 +13,32 @@ spec:
   subresources:
     status: {}
   versions:
-  - name: v1
-    served: true
-    storage: true
+    - name: v1
+      served: true
+      storage: true
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            managementState:
+              pattern: ^(Managed|Unmanaged|Removed|Forced)$
+              type: string
+              description: managementState indicates whether and how the operator
+                should manage the component
+            authentication:
+              properties:
+                logoutRedirect:
+                  pattern: ^((https):\/\/?)[^\s()<>]+(?:\([\w\d]+\)|([^[:punct:]\s]|\/?))$
+                  type: string
+                  description: Logout Redirect must be a valid URL
+            customization:
+              properties:
+                documentationBaseURL:
+                  pattern: ^((https):\/\/?)[^\s()<>]+(?:\([\w\d]+\)|([^[:punct:]\s]|\/?))\/$
+                  type: string
+                  description: Document base URL must be a valid URL that ends in a trailing slash
+                brand:
+                  pattern: ^(ocp|origin|okd|dedicated|online|azure)$
+                  type: string
+                  description: Brand must be be one of five values - ocp|origin|okd|dedicated|online
```

Comment:
`description` for brand also needs to be updated:

`description: Brand must be be one of five values - okd|ocp|origin|dedicated|online|azure`

I don't know if ordering matters... alphabetical vs upstream first vs?

Reply:
@zherman0 i think the only lingering is getting `azure` in the description here.

Reply:
@benjaminapetersen - fixed description - how does it look?